instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import Comment from './Comment';
+import './CommentSection.css';
+
+CommentSection.propTypes = {
+    postData: PropTypes.shape({
+        comments: PropTypes.array,
+        timestamp: PropTypes.string
+    })
+};
+
+class CommentSection extends React.Component {
+
+    state = {
+        comments: this.props.post.comments,
+        username: ""
```

Comment:
Since we don't have dynamic username's coming in, you could set the username to a default value like "Bob the Builder", that way a username will show up when you add a comment.